www/appsflyer.js

```diff
@@ -39,7 +39,15 @@ if(!window.CustomEvent) {
         var data = conversionData,
             event;
         if (typeof data === "string") {
-            data = JSON.parse(conversionData);
+            try {
```

Comment:
aktualizacja może narodzić kolejne problemy, a tego nie udało się zreprodukować. wolę zrozumieć najpierw problem, jak go zrozumiemy i  zreprodukujemy to będziemy mogli  sprawdzić potem ewentualnie jako kolejny krok - może aktualizować.